mewarpx/mewarpx/mwxrun.py

```diff
@@ -285,10 +290,10 @@ def get_gathered_rho_grid(self, species_name=None, include_ghosts=True):
             _libwarpx.libwarpx.warpx_depositRhoSpecies(
                 ctypes.c_char_p(species_name.encode('utf-8'))
             )
-        return _libwarpx._get_mesh_field_list(
-            _libwarpx.libwarpx.warpx_getGatheredChargeDensityFP,
-            self.lev, None, include_ghosts
-        )
+        if include_ghosts:
+            return self.rho_wrapper_ghosts[Ellipsis]
```

Comment:
What's the difference between returning `self.rho_wrapper_ghosts[Ellipsis]` and `self.rho_wrapper_ghosts`, won't both just return the full array?

Reply:
Nevermind, after looking through fields.py, the multifab wrapper object that is returned has specific behavior with Ellipsis that converts it to a numpy array.